docs/en/03_Upgrading/05_PHPUnit9.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrading to PHPUnit 9.5 for PHP8 support
+summary: Guidance on upgrading your project or module to use PHPUnit 9
+---
+
+# Upgrading to PHPUnit 9.5 for PHP8 support
+
+Unit testing in Silverstripe is done with the `SapphireTest` class which is a wrapper on top of the phpunit/phpunit `TestCase` class. Silverstripe CMS 4.9 and earlier used a forked version of PHPUnit 5.7 and was not compatible with PHP 8.
+
+Silverstripe CMS 4.10 onwards now supports PHPUnit 9.5 allowing projects to use PHP 8.
+
+## Dual support for PHPUnit 5.7 and PHPUnit 9.5
+
+`SapphireTest` now has dual support for both PHPUnit 5.7 and PHPUnit 9.5. This allows existing projects to upgrade to Silverstripe CMS 4.10 without breaking their unit-test suite, however their will be limited to PHP 7.3 and PHP 7.4.
+
+Support for PHPUnit 5.7 will continue until approximately January 2023 when PHPUnit 5.7 support will be dropped.
+
+PHPUnit versions 6.x, 7.x and 8.x are NOT supported.
+
+## How to enable support for PHPUnit 9.5
+
+Add `"phpunit/phpunit": "^9.5"` to the `"require-dev"` section of composer.json and run `composer update`.
+
+If you are upgrading a module rather than a website, ensure there's a specific requirement for `silverstripe/framework`: `^4.10` as Silverstripe CMS 4.9 and earlier are not compatible with PHPUnit 9.
+
+If the `"require"` block in composer.json does not have a requirement for `"silverstripe/framework"`, you can put  the requirement in `"require-dev"` so that it's only required when running CI or running unit tests locally. This will allow older versions of Silverstripe CMS to use the latest version of your module.
```

Comment:
```suggestion
If the `"require"` block in composer.json does not have a requirement for `"silverstripe/framework"`, you can put the requirement in `"require-dev"` so that it's only required when running CI or running unit tests locally. This will allow older versions of Silverstripe CMS to use the latest version of your module.
```

Reply:
Updated